pkgs/applications/office/onlyoffice-bin/default.nix

```
@@ -147,18 +147,20 @@ stdenv.mkDerivation rec {
 
     ln -s $out/share/desktopeditors/DesktopEditors $out/bin/DesktopEditors
 
-    wrapProgram $out/bin/DesktopEditors \
-        --set QT_XKB_CONFIG_ROOT ${xkeyboard_config}/share/X11/xkb \
-        --set QTCOMPOSE ${xorg.libX11.out}/share/X11/locale
-
     substituteInPlace $out/share/applications/onlyoffice-desktopeditors.desktop \
       --replace "/usr/bin/onlyoffice-desktopeditor" "$out/bin/DesktopEditor"
 
     runHook postInstall
   '';
 
   preFixup = ''
-    gappsWrapperArgs+=(--prefix LD_LIBRARY_PATH : "${runtimeLibs}" )
+    gappsWrapperArgs+=(
+      --prefix LD_LIBRARY_PATH : "${runtimeLibs}" \
+      --set QT_XKB_CONFIG_ROOT ${xkeyboard_config}/share/X11/xkb \
```

While we're at it, might as well quote it for shell scripting properness:

```suggestion
      --set QT_XKB_CONFIG_ROOT "${xkeyboard_config}/share/X11/xkb" \
```